unit_test/live_model_test.cc

```diff
@@ -796,11 +866,42 @@ BOOST_AUTO_TEST_CASE(ccb_explore_only_mode) {
   ++it;
 }
 
+BOOST_AUTO_TEST_CASE(ccb_explore_only_mode_multi_slot_response_detailed) {
+	u::configuration config;
+	cfg::create_from_json(JSON_CFG, config);
+	config.set(r::name::PROTOCOL_VERSION, "2");
+	config.set(r::name::EH_TEST, "true");
+	config.set(r::name::MODEL_SRC, r::value::NO_MODEL_DATA);
+	config.set(r::name::OBSERVATION_SENDER_IMPLEMENTATION, r::value::OBSERVATION_FILE_SENDER);
+	config.set(r::name::INTERACTION_SENDER_IMPLEMENTATION, r::value::INTERACTION_FILE_SENDER);
+	config.set(r::name::INTERACTION_FILE_NAME, "interaction.txt");
+	config.set(r::name::OBSERVATION_FILE_NAME, "observation.txt");
+	config.set(r::name::MODEL_VW_INITIAL_COMMAND_LINE, "--ccb_explore_adf --json --quiet --epsilon 0.0 --first_only --id N/A");
+
+	r::api_status status;
+	r::live_model model(config);
+	BOOST_CHECK_EQUAL(model.init(&status), err::success);
+
+	r::multi_slot_response_detailed response;
+	BOOST_CHECK_EQUAL(model.request_multi_slot_decision(JSON_CCB_CONTEXT, response), err::success);
+
+	BOOST_CHECK(strcmp(response.get_model_id(), "N/A") == 0);
+	BOOST_CHECK_EQUAL(response.size(), 2);
+
+
+	for (const auto& s : response) {
+		size_t action_id;
+		s.get_chosen_action_id(action_id);
+		BOOST_CHECK_EQUAL(action_id, 0);
+	}
+}
+
 const auto JSON_SLATES_CONTEXT = R"({"GUser":{"id":"a","major":"eng","hobby":"hiking"},"_multi":[{"TAction":{"a1":"f1"},"_slot_id":0},{"TAction":{"a2":"f2"},"_slot_id":0},{"TAction":{"a3":"f3"},"_slot_id":1},{"TAction":{"a4":"f4"},"_slot_id":1},{"TAction":{"a5":"f5"},"_slot_id":1}],"_slots":[{"Slot":{"a1":"f1"}},{"Slot":{"a2":"f2"}}]})";
 
 BOOST_AUTO_TEST_CASE(slates_explore_only_mode) {
   u::configuration config;
   cfg::create_from_json(JSON_CFG, config);
+  config.set(r::name::PROTOCOL_VERSION, "2");
```

Comment:
does this mean that we don't test slates for v1 anymore?

Reply:
That was added by mistake. Thanks for catching that